lib/src/component_declaration/component_base.dart

```diff
@@ -552,11 +552,6 @@ abstract class UiProps extends MapBase
       // https://github.com/dart-lang/sdk/issues/29904
       // Should have the benefit of better performance;
       return factory.build(props, childArguments);
```

Comment:
Should there still be an `if` surrounding this if we no longer have the `else`?

Reply:
Oh, actually yeah. The  `componentFactory` getter now always returns a `ReactComponentFactoryProxy`. Will remove.